fearless/Modules/Staking/StakingAmount/ViewModel/RewardDestinationViewModelFactory.swift

```diff
@@ -72,21 +72,24 @@ final class RewardDestinationViewModelFactory: RewardDestinationViewModelFactory
     ) -> LocalizableResource<RewardDestinationViewModelProtocol> {
         LocalizableResource { locale in
             let amountFormatter = amountFormatter.value(for: locale)
-            let percentageFormatter = NumberFormatter.percent.localizableResource().value(for: locale)
+            let percentageAPYFormatter = NumberFormatter.percentAPY.localizableResource().value(for: locale)
+            let percentageAPRFormatter = NumberFormatter.percentAPR.localizableResource().value(for: locale)
+
+            let zeroAmount = amountFormatter.string(from: 0.0) ?? "0"
```

Comment:
In the view we already handle empty state by assigning zeros. Why do we need it also here?

Reply:
Yes, we handle empty state, but here we try to handle case when `string(from:)` returns nil. Moreover, `DestinationReturnViewModel` does not accept Optionals and if we make zeroAmount Optional, the next code would look absolutely terrible:
`let rewardViewModel = DestinationReturnViewModel(
    restakeAmount: restakeAmount ?? zeroAmount,
    restakePercentage: restakePercentage ?? "",
    payoutAmount: payoutAmount ?? zeroAmount,
    payoutPercentage: payoutPercentage ?? ""
)`

Reply:
Got it. Looks like we need an empty state for it like shimmering. Will discuss it later